modules/openy_features/openy_location/modules/openy_loc_branch/openy_loc_branch.features.yml

```diff
@@ -1 +1,3 @@
+excluded:
+  - core.entity_view_mode.node.class_location
```

Comment:
I'll remove. This is from Features, when you uncheck something that Features is atempting to automatically include. It will not happen if you've already included an element in another feature an occurred here because of the order I created/update the features.